src/components/content/unread-records-notification.tsx

```diff
@@ -24,14 +28,50 @@ export const UnreadRecordsNotification = ({
   resources = defaultZAPTabs,
   text,
 }: UnreadRecordsNotificationProps) => {
+  const unreadRecordsMap = useUnreadRecordsByResource(resources);
+
+  for (const isUnread of unreadRecordsMap.values()) {
+    if (isUnread) {
+      return <UnreadNotificationIcon className={className} text={text} />;
+    }
+  }
+
+  return null;
+};
+
+// Provides the number of ZAP resources with unread notifications
+export function useUnreadZAPTabsCount(resources = defaultZAPTabs): number {
+  const unreadRecordsMap = useUnreadRecordsByResource(resources);
+  return [...unreadRecordsMap.entries()].reduce((count, hasUnread) => {
+    return hasUnread ? count + 1 : count;
+  }, 0);
```

Comment:
This should be the linted version
```suggestion
  return [...unreadRecordsMap.values()].reduce(
    (count, hasUnread) => (hasUnread ? count + 1 : count),
    0
  );
```